python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/aaa-accounting.j2

```diff
@@ -52,7 +52,24 @@
 {%     if aaa_accounting.system.default.type is arista.avd.defined and aaa_accounting.system.default.group is arista.avd.defined %}
 aaa accounting system default {{ aaa_accounting.system.default.type }} group {{ aaa_accounting.system.default.group }}
 {%     endif %}
-{%     if aaa_accounting.dot1x.default.type is arista.avd.defined and aaa_accounting.dot1x.default.group is arista.avd.defined %}
+{%     if aaa_accounting.dot1x.default.methods is arista.avd.defined %}
+{%         set accounting_default_list = [] %}
+{%         for method in aaa_accounting.dot1x.default.methods %}
+{%             if method.method == "logging" %}
+{%                 do accounting_default_list.append(method.method) %}
+{%             elif method.method == "group" and method.group is arista.avd.defined %}
+{%                 set group_cli = "group " ~ method.group %}
+{%                 if method.multicast is arista.avd.defined %}
+{%                     set group_cli = group_cli ~ " multicast" %}
+{%                 endif %}
+{%                 do accounting_default_list.append(group_cli) %}
+{%             endif %}
+{%         endfor %}
+{%         set accounting_list_cli = " ".join(accounting_default_list) %}
+{%         if aaa_accounting.dot1x.default.type is arista.avd.defined %}
```

Comment:
this should be check along with the `aaa_accounting.dot1x.default.methods`.

Reply:
Done.